Controllers/WordController.cs

```diff
@@ -0,0 +1,40 @@
+using Microsoft.AspNetCore.Mvc;
+using RandomWords.Service;
+
+namespace RandomWords.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class WordController
+{
+    private WordService wordService;
+    public WordController(WordService service)
+    {
+        this.wordService = service;
+    }
+    
+    [HttpGet("/words")]
+    public List<string> GetAllWords()
+    {
+        var list = this.wordService.GetAllWords();
+        return list;
+    }
+    
+    [HttpGet("/oneword")]
+    public string GetWord(int index)

```

Comment:
The parameter is not reflected in the Route.
```suggestion
    [HttpGet("/oneword/{index}")]
    public string GetWord(int index)

```